hazelcast/include/hazelcast/hzc.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
```

Comment:
I did not understand the usage of this file. It is not included anywhere as I see.

Reply:
By mistaken.

Fixed.